extras/controllerClient/Examples/example_csharp/SpeechPriority.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright(C) 2017-2023 NV Access Limited, Arnold Loubriat, Leonard de Ruijter
+// This file is covered by the GNU Lesser General Public License, version 2.1.
+// See the file license.txt for more details.
+
+namespace NVAccess.NVDA
+{
+    public enum SpeechPriority

```

Comment:
it may be also worth referencing the python enum here, even though this is an example. I don't think a 2-way reference is needed, as it can be the reader's responsibility to check the other enums and ensure this example reflects them correctly.